code/player/TTTPlayer.Commands.Server.cs

```diff
@@ -22,7 +22,7 @@ public static void RespawnPlayer()
 
             TTTPlayer player = ConsoleSystem.Caller.Pawn as TTTPlayer;
 
-            if (!player.IsValid() || ConsoleSystem.Caller.GetScore<bool>("forcedspectator", false))
+            if (!player.IsValid() || ConsoleSystem.Caller.GetValue<bool>("forcedspectator", false))
```

Comment:
I wonder if we should hardcore these strings somewhere. `forcedspectator`

Reply:
Like in the event system? Could do so if we plan introducing more of these, currently, "forcedspectator" is the only one.